index.js

```diff
@@ -328,12 +326,26 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
     }
 }
 
-// predicate-based error filter
+/**
+ * Helper function to make api calls - this abstraction is used to
+ * simplify testing
+ *
+ * @param  {String}  url
+ * @returns  {Promise}
+ */
+FacebookInsightStream._apiCall = function (url) {
```

Comment:
we use underscore for internal functions only no?..

Reply:
This is used as an internal function, yes - it shouldn't be used outside of the module.